server/src/db/migrations/20240430184012_createReviewTable.cjs

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("reviews", (table) => {
+    table.bigIncrements("id")
+    table.string("title")
+    table.text("body").notNullable()
+    table.integer("starRating").notNullable().unsigned()
+    table
+      .bigInteger("userId")
+      .index()
+      .unsigned()
+      .notNullable()
+      .references("users.id")
+    table
+      .bigInteger("sandwichId")
+      .index()
+      .unsigned()
+      .notNullable()
+      .references("sandwiches.id")
+    table.timestamp("createdAt").notNullable().defaultTo(knex.fn.now())
+		table.timestamp("updatedAt").notNullable().defaultTo(knex.fn.now())
```

Comment:
uh oh! 
- indentation is weird here
- and if you are multilining your ids, you. should be consistent and multiline your timestamp columns as well. Just choose one or the other!